client/src/components/formHelpers/form-fields.tsx

```diff
@@ -94,7 +95,6 @@ function FormFields({ formFields, options }: FormFieldsProps): JSX.Element {
       {formFields
         .filter(formField => !ignored.includes(formField.name))
         .map(({ name, label }) => (
-          // TODO: verify if the value is always a string
           <Field key={`${name}-field`} name={name}>
```

Comment:
`value` is not always a string.

If an initial value isn't provided, `value` will be `undefined` until the user interacts with the field. So the value type should be set to:

```suggestion
          <Field<string | undefined> key={`${name}-field`} name={name}>
```

Then, we remove the `value as string` type cast. And update `nullOrWarning` to take `value: string | undefined` instead.